libraries/AC_AutoTune/AC_AutoTune.cpp

```diff
@@ -1232,6 +1249,9 @@ void AC_AutoTune::update_gcs(uint8_t message_id)
     case AUTOTUNE_MESSAGE_FAILED:
         gcs().send_text(MAV_SEVERITY_NOTICE,"AutoTune: Failed");
         break;
+    case AUTOTUNE_MESSAGE_TESTING:
+        gcs().send_text(MAV_SEVERITY_NOTICE,"AutoTune: Pilot Testing");
```

Comment:
I wonder if we could make this clearer....  "Pilot - please test new gains" isn't quite right, but that's kind of what we want happening at that point?

Reply:
Happy to change, but I'm ok with the wording.